learning_journal/scripts/initializedb.py

```diff
@@ -0,0 +1,71 @@
+import os
+import sys
+import transaction
+
+from pyramid.paster import (
+    get_appsettings,
+    setup_logging,
+)
+
+from pyramid.scripts.common import parse_vars
+
+from ..models.meta import Base
+from ..models import (
+    get_engine,
+    get_session_factory,
+    get_tm_session,
+)
+
+# from ..models import MyModel
+from ..models import Entry
+
+JOURNAL_ENTRIES = [
+    {
+        "title": "Day 12 Learning Journal",
+        "creation_date": "23 August 2016",
+        "body": "Sample body text for Day 12 Learning Journal.",
+    },
+    {
+        "title": "Another Learning Journal",
+        "creation_date": "22 August 2016",
+        "body": "Sample body text for Another Learning Journal.",
+    },
+    {
+        "title": "A Wild Third Entry Appears!",
+        "creation_date": "21 August 2016",
+        "body": "Sample body text for A Wild Third Entry Appears.",
+    },
+]
+
+
+def usage(argv):
+    cmd = os.path.basename(argv[0])
+    print('usage: %s <config_uri> [var=value]\n'
+          '(example: "%s development.ini")' % (cmd, cmd))
+    sys.exit(1)
+
+
+def main(argv=sys.argv):
+    if len(argv) < 2:
+        usage(argv)
+    config_uri = argv[1]
+    options = parse_vars(argv[2:])
+    setup_logging(config_uri)
+    settings = get_appsettings(config_uri, options=options)
+
+    engine = get_engine(settings)
+
+    # Base.metadata.drop_all(engine)
+    # Base.metadata(engine).commit()
+    # # the above line from http://pythoncentral.io/sqlalchemy-faqs/
+    # Base.metadata.create_all(engine)
+
+    session_factory = get_session_factory(engine)
+
+    with transaction.manager:
+        dbsession = get_tm_session(session_factory, transaction.manager)
+        for entry in JOURNAL_ENTRIES:
+            temp = Entry(title=entry["title"],
+                         creation_date=entry["creation_date"],
```

Comment:
When you fix your model to use a DateTime or Date type for the creation_date field, you'll need to update this so that it converts the strings you have stored above into the datetime objects you'll need.